tests/e2e/e2e_http_ingress_test.go

```diff
@@ -0,0 +1,135 @@
+package e2e
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chart/loader"
+	helmcli "helm.sh/helm/v3/pkg/cli"
+	"k8s.io/api/extensions/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+
+	. "github.com/openservicemesh/osm/tests/framework"
+)
+
+var _ = OSMDescribe("HTTP ingress",
+	OSMDescribeInfo{
+		Tier:   1,
```

Comment:
I see the value in making ingress Tier 1. E2e  tests are up to 17 minutes though, we will need to speed these up.